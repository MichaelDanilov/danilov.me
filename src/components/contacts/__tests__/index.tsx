import React from 'react';
import renderer from 'react-test-renderer';
import Contacts from '..';

import Languages from '../../../constants/languages';

describe('Contacts', () => {
  it('english version renders correctly', () => {
    const tree = renderer
      .create(<Contacts lang={Languages.english} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('russian version renders correctly', () => {
    const tree = renderer
      .create(<Contacts lang={Languages.russian} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
