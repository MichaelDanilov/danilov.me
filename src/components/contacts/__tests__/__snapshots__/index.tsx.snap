// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts english version renders correctly 1`] = `
<div
  className="sc-AxjAm hKhHEk"
>
  <ul
    className="sc-AxirZ gXlCIn"
  >
    <li
      className="sc-AxiKw xmlnn"
    >
      <span>
        Email
      </span>
       - 
      <a
        href="mailto:michael@danilov.me"
      >
        michael@danilov.me
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <span>
        Telegram
      </span>
       - 
      <a
        href="https://t.me/MichaelDanilov"
      >
        @MichaelDanilov
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <span>
        Twitter
      </span>
       - 
      <a
        href="https://twitter.com/MonkAlbino"
      >
        @MonkAlbino
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <a
        href="https://fb.me/michaeldanilov"
      >
        Facebook
      </a>
    </li>
  </ul>
</div>
`;

exports[`Contacts russian version renders correctly 1`] = `
<div
  className="sc-AxjAm hKhHEk"
>
  <ul
    className="sc-AxirZ gXlCIn"
  >
    <li
      className="sc-AxiKw xmlnn"
    >
      <span>
        Электронная почта
      </span>
       - 
      <a
        href="mailto:michael@danilov.me"
      >
        michael@danilov.me
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <a
        href="https://vk.me/michael.danilov"
      >
        ВКонтакте
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <span>
        Twitter
      </span>
       - 
      <a
        href="https://twitter.com/MonkAlbino"
      >
        @MonkAlbino
      </a>
    </li>
    <li
      className="sc-AxiKw xmlnn"
    >
      <a
        href="https://fb.me/michaeldanilov"
      >
        Facebook
      </a>
    </li>
  </ul>
</div>
`;
